src/librustc_mir/borrow_check/nll/type_check/mod.rs

```diff
@@ -1585,6 +1585,12 @@ impl MirPass for TypeckMir {
         let id = tcx.hir.as_local_node_id(def_id).unwrap();
         debug!("run_pass: {:?}", def_id);
 
+        // When NLL is enabled, the borrow checker runs the typeck
+        // itself, so we don't need this MIR pass anymore.
+        if tcx.sess.nll() {
```

Comment:
```
[00:12:34] error[E0599]: no method named `nll` found for type `&rustc::session::Session` in the current scope
[00:12:34]     --> librustc_mir/borrow_check/nll/type_check/mod.rs:1590:21
[00:12:34]      |
[00:12:34] 1590 |         if tcx.sess.nll() {
[00:12:34]      |                     ^^^
```

in the rollup

Reply:
It's removed by f787bddcff .

Reply:
Actually, that PR seems to be more rollup-breaking, I'll put this back in, sorry about that